pages/docs/data-modeling/attributes.mdx

```diff
@@ -56,3 +55,18 @@ SQL can reference other attributes by surrounding their `alias` in double curly
 ### Using joined columns in custom SQL
 
 See [Using Joined Columns in Custom SQL](./model-joins#using-joined-columns-in-custom-sql) for more info.
+
+## Attribute folders
+
+You can organize your attributes into folders to keep your data model tidy and easy to navigate.
```

Comment:
Nit: maybe just reference in both sections that this available is for both attributes AND measures, when I first started reading this I got confused bc the wording made me think this was an attributes-only feature